index.js

```diff
@@ -61,7 +61,14 @@ module.exports.parseString = function(st, max) {
                     events[event_count][k].value = getDate(params["VALUE"], value);
                 } else {
                     if(params["TZID"] !== undefined) {
+                      //Allow to parse as per GMT
                         events[event_count][k].value = getDate(undefined, value.concat('Z'));
+                        //If a locale exists, strip preceeding /
+                        if(params["TZID"].search(/\//) != -1){
```

Comment:
I am not sure this does exactly what you want it to do, it seems that this check will find _any_ `/` in the TZID then take a substring, so, for example, wouldn't this:

```ts
America/Los_Angeles
```

match for your `.search(...)` call? Which would then end up with `merica/Los_Angeles`?

I think you would just want to check if the string starts with `/`, which simplifies the call to `if(params["TZID"].startsWith('/'))`

Reply:
Yes agree re the search. I'll make the change to the code and commit the change. I'll also write a test case for it and submit. Thanks for the advice.

Reply:
Sounds good - I'm about to head on a trip for a bit over a week tomorrow so I may go quiet here, but I will try my best to come back to this promptly after my trip. I hope you have a workaround for the time being.